src/test/java/games/strategy/triplea/delegate/BattleTrackerTest.java

```diff
@@ -56,26 +55,27 @@ public void verifyRaidsWithNoBattles() {
 
   @Test
   public void verifyRaids() {
-    Territory territory = new Territory("terrName", mockGameData);
+    final Territory territory = new Territory("terrName", mockGameData);
     final Route route = new Route(territory);
-    PlayerID playerId = new PlayerID("name", mockGameData);
+    final PlayerID playerId = new PlayerID("name", mockGameData);
 
     // need at least one attacker for there to be considered a battle.
-    Unit unit = new TripleAUnit(new UnitType("unit", mockGameData), playerId, mockGameData);
+    final Unit unit = new TripleAUnit(new UnitType("unit", mockGameData), playerId, mockGameData);
     final List<Unit> attackers = Collections.singletonList(unit);
 
     when(mockDelegateBridge.getData()).thenReturn(mockGameData);
     when(mockGameData.getProperties()).thenReturn(mockGameProperties);
     when(mockGameData.getRelationshipTracker()).thenReturn(mockRelationshipTracker);
     when(mockGameProperties.get(Constants.RAIDS_MAY_BE_PRECEEDED_BY_AIR_BATTLES, false)).thenReturn(true);
-    when(mockGetBattleFunction.apply(territory, IBattle.BattleType.BOMBING_RAID)).thenReturn(mockBattle);
+    doReturn(null).when(mockGetBattleFunction).apply(territory, IBattle.BattleType.AIR_RAID);
+    doReturn(mockBattle).when(mockGetBattleFunction).apply(territory, IBattle.BattleType.BOMBING_RAID);
 
     // set up the testObj to have the bombing battle
     testObj.addBombingBattle(route, attackers, playerId, mockDelegateBridge, null, null);
 
     testObj.fightAirRaidsAndStrategicBombing(mockDelegateBridge, () -> Collections.singleton(territory),
         mockGetBattleFunction);
 
-    verify(mockBattle, times(1)).fight(mockDelegateBridge);
+    verify(mockBattle).fight(mockDelegateBridge);
```

Comment:
`times(1)` is the default value for the second argument to `verify()`, so it does not need to be explicitly provided.